packages/reports/build/constants.js

```diff
@@ -20,6 +20,7 @@ const {
 } = require('./loaders-config');
 
 const ALIASES = {
+    'react/jsx-runtime': 'react/jsx-runtime.js',
```

Comment:
what is this

Reply:
Need to add it to use deriv-com/ui library